package.json

```diff
@@ -122,6 +122,7 @@
     "babel-plugin-transform-rename-import": "2.3.0",
     "babel-preset-jest": "24.6.0",
     "browserslist": "4.6.3",
+    "bundlesize": "0.18.0",
```

Comment:
We also need to trigger the bundlesize on CI...

Reply:
how do we do this? I can't find instructions

Reply:
I added the token to our env vars

Reply:
Just add a step on CI to execute `bundlesize`. That's it.

After this: https://github.com/commercetools/ui-kit/blob/20cd7a4f5a2b2b75c6e9052712631253a98a5c6b/.circleci/config.yml#L36

Just add 

```
- run: yarn run bundlesize
```

Reply:
done